loader.js

```diff
@@ -6,14 +6,6 @@
 const {compile} = require('carmi');
 const esm = require('esm');
 
-const clearAllModulesLoadedForCarmi = requiredPreCarmi => {
-  Object.keys(require.cache).forEach(key => {
-    if (!requiredPreCarmi.has(key)) {
-      delete require.cache[key];
-    }
-  });
-};
-
 module.exports = function CarmiLoader() {
   this.cacheable(false);
```

Comment:
can it be removed after adding the dependencies?

Reply:
Thanks 👍